howto/stream/oob-data.md

```diff
@@ -1,14 +1,34 @@
-The exchange of out-of-band data transmission between Android and Anbox (WebRTC platform) enables Android application developers to trigger an action from an Android application and forward it to Anbox(WebRTC platform). Once Anbox receives the action, as one peer of WebRTC platform, the action will be propagated from the Anbox to the remote peer(WebRTC client) through WebRTC data channel so that client can react to the action received from the remote peer and respond accordingly on the UI.
+Enabling out-of-band (OOB) data transmission between an Android application and a WebRTC client makes it possible to exchange data and trigger actions between an Android application and a WebRTC client.
 
-This document will walk you through how to send a message from an Android application
-running in the container to the client application developed with the Anbox Streaming
+Anbox Cloud provides two versions of this OOB data exchange:
+
+- [Version 1](#oob-v1) enables Android application developers to trigger an action from an Android application and forward it to a WebRTC client through the Anbox WebRTC platform. When Anbox receives the action, as one peer of the WebRTC platform, the action is propagated from Anbox to the remote peer (the WebRTC client) through a WebRTC data channel. The client can then react to the action received from the remote peer and respond accordingly on the UI.
+
+  This version supports only half-duplex data transmission. It allows sending data from an Android application to a WebRTC platform, but it is not possible to receive data from the WebRTC client to an Android application.
+- [Version 2](#oob-v2), which is currently in a draft state, provides a full-duplex bidirectional data transmission mode in which data can flow in both directions at the same time.
```

Comment:
Hmm, the `draft state` may not accurately describe the current status of OOB v2. The design doc is approved and there shouldn't be changes to the interface. Even without `Anbox Data Proxy` service, at the current stage, people already can make use of the exposed Unix domain sockets and set up the proxy tunnel to forward data to both ends. The `Anbox Data Proxy` is just another component (more like an addon) sitting next to Anbox WebRTC platform, facilitating developers to do the data exchange between Android and Anbox WebRTC platform. When we implement it in 1.16.0, the current existing OOB v2 implementation won't change at all.
I prefer to drop `which is currently in a draft state,` in this sentence. @morphis what do you think?

Reply:
Reworded this a bit, and changed the order of version 1 and version 2.